documentation/design-docs/dotnet-tools.md

```diff
@@ -551,6 +551,53 @@ UNINSTALL
       Uninstalling SOS from ~/.dotnet/sos
       Complete
 
+### dotnet-gcheapdump
```

Comment:
In the past I've always seen it called 'gcdump', did something lead you to 'gcheapdump' instead?

Reply:
I honestly can't think of anything in particular that lead me to that name.  I think `gcdump` looks a little cleaner, so I'll go ahead and change everything to that 😄.